Formula/apt.rb

```diff
@@ -0,0 +1,114 @@
+class Apt < Formula
+  desc "Advanced Package Tool"
+  homepage "https://wiki.debian.org/apt"
+  url "https://deb.debian.org/debian/pool/main/a/apt/apt_2.3.7.tar.xz"
+  sha256 "c946a126263b3ea51a15216689602367c0da95d6878b76dc891ea335d87bd936"
+  license "GPL-2.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "doxygen" => :build
+  depends_on "libxslt" => :build
+  depends_on "perl" => :build
```

Comment:
Does `apt` not need brewed `perl` at runtime?

Also related, are the installed Perl resources needed at runtime? We usually create env wrapper scripts to add `PERL5LIB` so that installed Perl modules are found.

If Perl resources are only used at build time, we should probably install them into `buildpath` rather than `libexec`.

Reply:
The `triehash` executable it installs does need `perl`, so I've changed that to be a runtime dependency.  I also had it install the Perl dependencies to `buildpath`.

Reply:
Do we need to distribute `triehash` executable?
Also, I think `triehash` should run using Linux system Perl (unless we overwrite the shebang `#!/usr/bin/perl -w`) so probably fine to keep `perl` build-time unless there is something else that needs brewed Perl.

It isn't part of Debian package: https://packages.debian.org/sid/amd64/apt/filelist
And AUR claims it is build-time: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=apt#n15

Reply:
It looks like a build-time command run in CMake `apt-pkg/CMakeLists.txt`
```cmake
execute_process(COMMAND ${TRIEHASH_EXECUTABLE}
                        --ignore-case
                         --header ${PROJECT_BINARY_DIR}/include/apt-pkg/tagfile-keys.h
                         --code ${CMAKE_CURRENT_BINARY_DIR}/tagfile-keys.cc
                         --enum-class
                         --enum-name pkgTagSection::Key
                         --function-name pkgTagHash
                         --include "<apt-pkg/tagfile.h>"
                         ${CMAKE_CURRENT_SOURCE_DIR}/tagfile-keys.list)
```